app/config/AppConfig.scala

```diff
@@ -34,4 +35,10 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   val eisgbUrl: String             = eisBaseUrl ++ config.get[String]("microservice.services.eis.gb.uri")
   val eisgbBearerToken: String     = config.get[String]("microservice.services.eis.gb.headers.bearerToken")
 
+  val apiXiRouting: RoutingOption = RoutingOption.parseRoutingOption(config.getOptional[String]("microservice.features.enableAPIXIRouting").getOrElse("Xi"))
```

Comment:
Could we make these non-optional?  I don't like the idea that we accidentally route somewhere because we haven't set a config value; I'd much prefer the application to fail in that scenario

Reply:
I'll discuss this with @crystalmark because this approach was to ensure there was a defaulted value. I'll make sure to update you.

Reply:
Cool, thank you.  The distinction is that if we have them optional and supply a default value, the default value is hardcoded in the application - i.e. it isn't configurable.  Having this mandatory means a value must be supplied (or else the app won't even start), but it has to be supplied via configuration.

In the former, you could very easily for example mis-type a config key name and think you're controlling it via config, but accidentally be using the hardcoded value.  In the latter case that's impossible.